src/graphql/schema/Tick.gql

```diff
@@ -120,6 +120,40 @@ enum TickSource {
   MP
 }
 
+"""The type of attempt the user wants to tick for a route."""
+enum TickAttemptType {
+  "Onsight: Sending a route fist try with no prior knowledge"
+  Onsight
+  "Flash: Sending a route first try with prior knowledge"
+  Flash
+  "Pinkpoint: Sending a route with no falls on pre-placed gear"
+  Pinkpoint
+  "French Free: Doing the route with pulling on gear or draws"
+  Frenchfree
+  "Attempt"
+  Attempt
+  "Send: Sending a route with no falls"
```

Comment:
I'm realizing we need "fell/hung" as a separate value. I was originally planning on treating "attempt" as "fell/hung" for roped climbs, but we don't always have a way to tell if the climb was a roped climbed or a boulder. ie: if the tick was imported from MP but doesn't have a corresponding entry in our DB. I think we should add back "fell/hung" to support this

Reply:
After discussion at sprint planning, we are NOT going to use "Fell/Hung". I've updated the logic in the MP import code https://github.com/OpenBeta/open-tacos/pull/1259 to handle this well, and I've written up a markdown document about tick logic, since it's complicated.  @PrithipalKhalsa   I had to make a couple changes to your code to get MP imports working along with PR 1259. Would you mind cherry-picking this commit https://github.com/glassbead0/openbeta-graphql/commit/1341fd5fdfa2962236440a8bc884edb798138a6c
into this PR? Thanks! (that commit also has the markdown doc)